### PR TITLE
Now using the flutterlookup location also for all commands

### PR DESCRIPTION
**Pull request by Kavantix**
This pull requests adds the ability to fully change the flutter sdk location such that all commands will correctly use it.
This can be very useful if one want to run a specific project on another channel of flutter

### CODE REVIEW

File: src/lib/sdk.ts [hunk 1]
```diff
@@ -110,10 +116,14 @@ class FlutterSDK {
 
       if (flutterPath) {
         flutterPath = await getRealPath(flutterPath);
+        flutterPath = flutterPath.trim();
         if (flutterPath.endsWith(join('bin', 'flutter'))) {
-          this._sdkHOme = flutterPath.replace(join('bin', 'flutter'), '');
+          this._sdkHome = flutterPath.replace(join('bin', 'flutter'), '');
+        } else {
+          this._sdkHome = flutterPath;
```
Comment (iamcco) [inline src/lib/sdk.ts:123]: `this._sdkHome = flutterPath` flutterPath is flutter command path, so it seems can't be sdkHome.

File: src/lib/sdk.ts [hunk 2]
```diff
@@ -110,10 +116,14 @@ class FlutterSDK {
...
         }
-        log(`flutter command path => ${flutterPath}`);
+        this._flutterCommand = join(this._sdkHome, 'bin', 'flutter');
```
Comment (iamcco) [inline src/lib/sdk.ts:125]: `this._flutterCommand = join(this._sdkHome, 'bin', 'flutter');`  can be just `this._flutterCommand = flutterPath` ?

File: src/lib/sdk.ts (hunk 1 above)
Comment (Kavantix) [reply src/lib/sdk.ts:123]: Well I assumed here that if it did not end with `/bin/flutter` that the user gave use the sdk directory.
If you think we should not do that we should probably throw an error to the user that the given flutter lookup is incorrect

File: src/lib/sdk.ts (hunk 2 above)
Comment (Kavantix) [reply src/lib/sdk.ts:125]: This depends on your answer on the previous comment ;)

File: src/lib/sdk.ts (hunk 1 above)
Comment (iamcco) [reply src/lib/sdk.ts:123]: OK, got it.

**Comment by iamcco** [comment]
Sorry for the late response, just back from Chinese New Year vacation

**Pull request merged**